.github/workflows/release.yml

```diff
@@ -0,0 +1,146 @@
+name: Release
+
+on:
+  workflow_dispatch:
```

Comment:
*nit/non-blocking* We should only release if unit tests have passed

Reply:
I rewrote the whole unit pipeline, but I considered adding it via a separate PR.